### ISSUE TITLE
extract.sh location

### ISSUE THREAD

**Issue by mmerrittsmith** [mansueto-institute/prclz#14]
What do you think makes sense for the location of the extract script? I would argue that currently not placing it anywhere in  package or providing it after downloading a file from geofabrik doesn't make sense because then you have to figure out where it got downloaded to. It might make more sense to either:

A. Include it in a newly created directory after the user runs download with geofabrik
B. Communicate to the user where the script can be found either locally (if we choose to automatically download it) or online (if we don't) after they run download with geofabrik

**Comment by satejsoman** [comment]
my preference is the former, and i think we can make it even more seamless than option A by:
- having `setup.py` copy the extract script to the package install location ([like we do with the country codes](https://github.com/mansueto-institute/prclz/blob/master/setup.py#L17)), 
- and then either:  
  - having cli.py call the script in a subprocess when the user does something like `prclz extract`, or 
  - just add the script to the user's path like the main cli entry point

(I've punted on making a windows version but we'll probably need that at some point too)

what do you think? i don't think what i've laid out is mutually exclusive with either option you suggest.

**Comment by mmerrittsmith** [comment]
Yeah that's cool with me. I think prclz extract makes sense, that way we're giving them a Python wrapper for something that's currently an external shell script. We will need an equivalent Windows version, but I'm not sure I'm proficient enough at batch to be the one to do it. I'll look into it though.
